va/nfv/ovs-dpdk/kustomization.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: kustomize.config.k8s.io/v1alpha1
+kind: Component
+
+transformers:
+# Set namespace to OpenStack on all namespaced objects without a namespace
+- |-
+  apiVersion: builtin
+  kind: NamespaceTransformer
+  metadata:
+    name: _ignored_
+    namespace: openstack
+  setRoleBindingSubjects: none
+  unsetOnly: true
+  fieldSpecs:
+  - path: metadata/name
+    kind: Namespace
+    create: true
+
+components:
+- ../../../lib/deploy
+- ../../../lib/nncp
```

Comment:
These two lines are out of date. Please rebase and apply the changes.

https://github.com/Jaganathancse/architecture/commit/e313fb6320607c6d673b3c5878e2c02ab44a8bcf

Reply:
This components list is out of date. It should only have the following:
```
components:
- ../../../lib/networking
- ../../../lib/control-plane
```

For example please see this:

https://github.com/openstack-k8s-operators/architecture/blob/main/va/nfv/sriov/kustomization.yaml#L19-L21